settings/FixedDueDateScheduleDetail.js

```diff
@@ -97,7 +96,7 @@ class FixedDueDateScheduleDetail extends React.Component {
           open={sections.generalInformation}
           id="generalInformation"
           onToggle={this.handleSectionToggle}
-          label="General information"
+          label={formatMessage({ id: 'ui-circulation.settings.fDDSform.about' })}
```

Comment:
Please use \<FormattedMessage\> here as it is now not wrapped in span and can be safely used.